apps/website/src/components/content/YouTube.tsx

```diff
@@ -143,33 +131,25 @@ type LightboxCtxProps = {
 
 type LightboxProps = {
   id?: string;
-  onInit?: () => void;
-  onDestroy?: () => void;
   className?: string;
   children: ReactNode | ((ctx: LightboxCtxProps) => ReactNode);
+  defaultOpen?: number;
```

Comment:
maybe there is another name. its not a controlled value so i tried to use the defaultX naming convention like its common for html elements?!

Reply:
I very much am not a fan of non-stateful props like this -- if we wanted to drive this by a prop, I'd probably want an `open` prop as well as an `onChange`, and expect them to always be in sync. If we just want something we use on demand, it should be a method not a prop imo.

Reply:
sure. that should be quite possible to make it synced/controlled